### DISCUSSION TITLE
Is PyG compatible with TorchServe?

### DISCUSSION THREAD

**Discussion by levulinh** [pyg-team/pytorch_geometric#3194] [Q&A]
Like the title, has anyone tried to deploy PyG models using TorchServe?
Sorry for asking before trying but I am kind of in between too many things for my thesis now.
Thank you guys in advance.

**Comment by rusty1s** [comment]
I personally haven't, but I'm eager to help in case you stumble upon any issues. Please let me know how it goes.
